R/sensorreading-data.R

```diff
@@ -47,7 +47,7 @@ formatSensorData <- function(data, columnNames){
     listElements <- data[listRows,]
     
     if ("displayTime" %in% names(data)) {
-      if(!is.na(listElements$displayTime) || is.null(listElements$time)) {
+      if(!isEmpty(listElements$displayTime) || isEmpty(listElements$time)) {
```

Comment:
not sure this is doing what we would expect...especially for listElements$time. There might be a case where we can continue if its NA, but not NULL? Hence, it was only checking for NULL, but isEmpty checks for both. Which we had tests that looked at this unique condition...